static/js/protein/chart.ts

```diff
@@ -219,6 +240,76 @@ function addYLabel(
     .text(labelText);
 }
 
+export function drawTissueLegend(id: string, data: ProteinStrData[]): void {
+  // checks if the data is empty or not
+  if (_.isEmpty(data)) {
+    return;
+  }
+  const svg = d3
+    .select("#tissue-score-legend")
+    .append("svg")
+    .attr("width", GRAPH_WIDTH_XL)
+    .attr("height", GRAPH_HEIGHT_XS);
+  const data0 = d3.keys(TISSUE_LEGEND_DICT);
+  // slicing the dictionary in half to display the legend in two rows
+  const data1 = data0.slice(0, 8);
```

Comment:
instead of using 8 and 15, should calculate those indices using the length of data0 instead so in the future, if that object gets updated, you don't have to also update these numbers.

nit: name as dataRowOne and dataRowTwo instead (if name of constant at line 253 is updated, then <name of constant>RowOne and <name of constant>RowTwo)

Reply:
Done!